modules/programs/vdirsyncer-accounts.nix

```diff
@@ -93,9 +93,9 @@ in {
     };
 
     verify = mkOption {
-      type = types.nullOr types.bool;
+      type = types.nullOr types.path;
       default = null;
-      description = "Verify SSL certificate.";
```

Comment:
Could you add the example from http://vdirsyncer.pimutils.org/en/stable/ssl-tutorial.html#custom-root-cas ?

Reply:
Added.